conan/tools/cmake/cmake.py

```diff
@@ -199,16 +214,24 @@ def install(self, build_type=None, component=None, cli_args=None, stdout=None, s
                            not build type.
         :param cli_args: A list of arguments ``[arg1, arg2, ...]`` for the underlying build system
                          that will be passed to the command line: ``cmake --install ... arg1 arg2``
+        :param build_subfolder: The name of a subfolder to be created inside the ``build_folder``
```

Comment:
```suggestion
        :param build_subfolder: (Experimental) The name of a subfolder to be created inside the ``build_folder``
```

Quick question, if it is necessary to define the same argument for the three configure+build+install, and it will fail if different, maybe it makes sense to define it in the constructor once?